### PR TITLE
Httpclient change

### PR DESCRIPTION
**Pull request by amit-tarento**

### CODE REVIEW

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/pom.xml [hunk 1]
```diff
@@ -154,7 +154,7 @@
 		<dependency>
 			<groupId>com.googlecode.libphonenumber</groupId>
 			<artifactId>libphonenumber</artifactId>
-			<version>8.10.2</version>
+			<version>8.12.5</version>
```
Comment (indrajra) [inline actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/pom.xml:157]: Can you check this library change and its significance?

File: actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java [hunk 2]
```diff
@@ -177,7 +177,7 @@ private void cacheFrameworkFieldsConfig() {
   private void updateUser(Request actorMessage) {
     Util.initializeContext(actorMessage, TelemetryEnvKey.USER);
     actorMessage.toLower();
-    Util.getUserProfileConfig(systemSettingActorRef);
+    //Util.getUserProfileConfig(systemSettingActorRef);
```
Comment (indrajra) [inline actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java:180]: sure?

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/test/java/org/sunbird/common/models/util/fcm/FCMNotificationTest.java [hunk 3]
```diff
@@ -41,15 +42,16 @@
...
-  HttpUtil.class,
+  HttpClientUtil.class,
   System.class,
   Notification.class
 })
 @PowerMockIgnore({"javax.management.*", "javax.net.ssl.*", "javax.security.*"})
 public class FCMNotificationTest {
 
-  @Test
+  //@Test
```
Comment (indrajra) [inline actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/test/java/org/sunbird/common/models/util/fcm/FCMNotificationTest.java:52]: Shouldn't these be in uncommented state? Pls add description so as to aid in review.

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/pom.xml (hunk 1 above)
Comment (amit-tarento) [reply actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/pom.xml:157]: to validate the new phone number, as this library validating the phone number based on area code.

File: actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java (hunk 2 above)
Comment (amit-tarento) [reply actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java:180]: commented as this line of code just reading the data from system setting table but not using that

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/test/java/org/sunbird/common/models/util/fcm/FCMNotificationTest.java (hunk 3 above)
Comment (amit-tarento) [reply actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/test/java/org/sunbird/common/models/util/fcm/FCMNotificationTest.java]: uncommented the test case

**Pull request merged**